### PR TITLE
Scaling added to PorousFlowRelativePermeability

### LINKED ISSUE #8886: PorousFlow task: include overall scaling in relative permeabilities
### Description of the enhancement or error report

Include an overall scaling parameter in relative permeability relations

### Rationale for the enhancement or information for reproducing the error

Currently relperm = 1 when saturation = 1.   For single-phase scenarios this is appropriate, but for multiphase scenarios it is not necessarily true that the relperm for each phase is unity when that phase's saturation is unity.  Therefore an overall scaling parameter is needed.

### Identified impact
(i.e. Internal object changes, limited interface changes, public API change, or a list of specific applications impacted)

Only PorousFlowRelativePermeabilityBase needs to be enhanced.

### PR DESCRIPTION
**Pull request by WilkAndy**
Fixes #8886

Scaling is useful in multiphase scenarios.

### CODE REVIEW

File: modules/porous_flow/src/materials/PorousFlowRelativePermeabilityBase.C [hunk 1]
```diff
@@ -12,6 +12,7 @@ InputParameters
 validParams<PorousFlowRelativePermeabilityBase>()
 {
   InputParameters params = validParams<PorousFlowMaterialBase>();
+  params.addParam<Real>("scaling", 1.0, "Relative permeability is multiplied by this factor");
```
Comment (cpgr) [inline modules/porous_flow/src/materials/PorousFlowRelativePermeabilityBase.C]: Hi Andy,
Should this be restricted to < 1?

Comment (WilkAndy) [inline modules/porous_flow/src/materials/PorousFlowRelativePermeabilityBase.C]: Nah, i kind of want scaling >= 0, for generality.  Eg, sometimes i suspect that the insitu permeability has been measured as the permeability for water, and then gas permeability might have scaling > 1

Comment (cpgr) [inline modules/porous_flow/src/materials/PorousFlowRelativePermeabilityBase.C]: No worries

Comment (cpgr) [inline modules/porous_flow/src/materials/PorousFlowRelativePermeabilityBase.C]: Although, maybe you should check that scaling is positive

**Review by permcody — APPROVED** [review]

**Comment by WilkAndy** [comment]
Thanks for the review and merge, @permcody ,

a

**Pull request merged**